cli/cmd/dashboard.go

```diff
@@ -60,5 +62,6 @@ func init() {
 
 	// This is identical to what `kubectl proxy --help` reports, `--port 0`
 	// indicates a random port.
-	dashboardCmd.PersistentFlags().IntVarP(&proxyPort, "port", "p", 8001, "The port on which to run the proxy. Set to 0 to pick a random port.")
+	dashboardCmd.PersistentFlags().IntVarP(&dashboardProxyPort, "port", "p", 8001, "The port on which to run the proxy. Set to 0 to pick a random port.")
+	dashboardCmd.PersistentFlags().BoolVar(&dashboardSkipBrowser, "no-browser", false, "Does not automatically open the default browser.")
```

Comment:
Can we call this flag `--url`? That's more consistent with the `minikube service` command:

```
      --url                Display the kubernetes service URL in the CLI instead of opening it in the default browser
```